bin/ch-test

```diff
@@ -186,18 +337,35 @@ fi
 
 # Ensure BATS_TMPDIR is set to /tmp (issue #278).
 if [[ -n $BATS_TMPDIR ]]; then
-    if [[ ! $BATS_TMPDIR == '/tmp/'* ]]; then
+    if [[ "$BATS_TMPDIR" != '/tmp'* ]]; then
```

Comment:
(1) You can skip the double quotes because it's Bash `[[ ]]`.
(2) This tests if `$BATS_TMPDIR` *begins with* `/tmp` rather than *is equal to* `/tmp`.